suite-native/atoms/src/Table.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactNode } from 'react';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type TableCell = string | ReactNode;
```

Comment:
nit: Should be enough like this. String is already part of ReactNode type

```suggestion
type TableCell = ReactNode;
```

Reply:
done